janus_core/calculations/geom_opt.py

```diff
@@ -301,6 +301,7 @@ def run(self) -> None:
         if self.logger:
             self.logger.info("After optimization spacegroup: %s", s_grp)
             self.logger.info("Max force: %.6f", max_force)
+            self.logger.info("Final energy: %.6f", self.struct.get_potential_energy())
```

Comment:
Just to double check this is "guaranteed" to not start a new calculation.

Reply:
also be sure you do not use a format one may want the precise energy

Reply:
> Just to double check this is "guaranteed" to not start a new calculation.

As long as energy is in the results dictionary (which it should be at this point), I believe so, from this function: https://gitlab.com/ase/ase/-/blob/master/ase/calculators/calculator.py?ref_type=heads#L515



> also be sure you do not use a format one may want the precise energy

Are you saying we don't want `.6f`?

Reply:
yes no format in summary

Reply:
This is in the log rather than the summary, so I think it needs some kind of formatting. We could do `%s` which I think basically keeps the full precision?

Would we want the same for the max force?

Reply:
if you are at it yes do the max force... yap whatever you like.